chaoscenter/web/src/controllers/InviteNewMembers/helper.ts

```diff
@@ -0,0 +1,17 @@
+import type { User } from '@api/auth';
+import type { InviteUserDetails } from './types';
+
+export function generateInviteUsersTableContent(userData: Array<User> | undefined): Array<InviteUserDetails> {
```

Comment:
These checks can be moved inline...this function will only confuse and this practice may not be followed by other devs if they don't know this function exists